hapi/utils.py

```diff
@@ -1,6 +1,6 @@
-import httplib
+import http.client
```

Comment:
`http` isn't a library in Python 2.  `httplib` was renamed to `http.client` in Python 3.  I'm not sure this will be compatible for 2 and 3.

Reply:
### Python 2 only:
import httplib
import Cookie
import cookielib
import BaseHTTPServer
import SimpleHTTPServer
import CGIHttpServer

### Python 2 and 3 (after ``pip install future``):
import http.client
import http.cookies
import http.cookiejar
import http.server`

from http://python-future.org/compatible_idioms.html